Extensions/DialogueTree/JsExtension.js

```diff
@@ -220,7 +220,7 @@ module.exports = {
         'JsPlatform/Extensions/yarn32.png'
       )
       .addParameter('string', _('State Variable Name'), '', false)
-      .addParameter('expression', _('Variable Value'), '', false)
+      .addParameter('string', _('Variable Value'), '', false)
```

Comment:
Is this done on purpose? This can break a fair number of games using the extension, no?

Reply:
yes it was, it should have been a string from the start, as the previous (expression) only allowed numbers - which is not at all compatible with what the yarn syntax and bondagejs supports (booleans, strings, floats and integers)

Reply:
This is also more compatible with how you set/get them inside yarn (always strings) and they are checked in bondagejs (parsed from the json file strings that get converted to their appropriate type)

Reply:
> I am fairly convinced nobody actually used this properly in their game, otherwise we would have seen a bug report to complain about those types not working

Ok fair enough!